src/View/Components/TwillFormComponent.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace A17\Twill\View\Components;
+
+use Illuminate\View\Component;
+
+abstract class TwillFormComponent extends Component
+{
+    public $name;
+    /**
+     * @var \A17\Twill\Models\Model
+     */
+    public $item;
+    public $label;
+    public $form_fields;
+
```

Comment:
Thinking of something to add here like:

getVuexStoreData

We could also go further and do something like:

getVueComponent
getVueComponentProps

and always render the same view file.